tests/AutoReview/ProjectCodeTest.php

```diff
@@ -129,7 +128,6 @@ final class ProjectCodeTest extends TestCase
     private static $nonTestedConcreteClasses = [
         ConfigureCommand::class,
         InfectionCommand::class,
-        SelfUpdateCommand::class,
```

Comment:
Note to self: The `padraic/phar-updater` dependency can also be removed now, fixing #425